Cargo.toml

```diff
@@ -8,7 +8,8 @@ members = ["bitwarden_license/*", "crates/*"]
 version = "1.0.0"
 authors = ["Bitwarden Inc"]
 edition = "2021"
-# Note: Changing rust-version should be considered a breaking change
+# Important: When updating rust-version, make sure to update the Readme file
```

Comment:
We don't actually seem to be mentioning the rust-version required in the readme file that I can see